### PR TITLE
Oracle Analytics Racing Challenge - Creation WMS ID: 11369

### PR DESCRIPTION
**Pull request by ThaisHenrique**
# Add here your WMS ID: 11369

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by bhenndricks — APPROVED** [review]

File: oci-analytics-challenge/workshops/sandbox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "workshoptitle": "Oracle Analytics Racing Challenge",
+    "help": "isabella.alvarez@oracle.com, breno.comin@oracle.com, thais.henrique@oracle.com",
```
Comment (anooshapilli) [inline oci-analytics-challenge/workshops/sandbox/manifest.json:3]: Update the help email to analytics stakeholder email - livelabs-help-analytics_us@oracle.com

File: oci-analytics-challenge/workshops/sandbox/manifest.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "variables": ["../../variables/variables.json",
+        "../../variables/variables-in-another-file.json"],
+    "tutorials": [
+        {
+            "title": "Introdução",
+            "description": "Bem-vindo ao Oracle Analytics Racing Challenge",
+            "filename": "../../intro/intro.md"
+        },
+        {
+            "title": "Pré-requisitos",
```
Comment (anooshapilli) [inline oci-analytics-challenge/workshops/sandbox/manifest.json:13]: Make this Lab 1 and update the rest of the lab numbers in this file. Add Get Started lab after introduction lab

**Review by anooshapilli — COMMENTED** [review]
@ThaisHenrique - Could you delete the ini file in the analytics-challenge images folder. Please make all the requested changes to approve the PR.

File: oci-analytics-challenge/workshops/sandbox/manifest.json (hunk 1 above)
Comment (ThaisHenrique) [reply oci-analytics-challenge/workshops/sandbox/manifest.json]: Done

File: oci-analytics-challenge/workshops/sandbox/manifest.json (hunk 2 above)
Comment (ThaisHenrique) [reply oci-analytics-challenge/workshops/sandbox/manifest.json:13]: This is the portuguese version of Get Started

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**